src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -113,48 +118,53 @@ public void setArticleDataDirectoryPath(Path articleDataDirectoryPath) {
     //=========== EntryBook ================================================================================
 
     @Override
-    public void setEntryBook(ReadOnlyEntryBook entryBook) {
-        this.entryBook.resetData(entryBook);
+    public void setListEntryBook(ReadOnlyEntryBook listEntryBook) {
+        this.listEntryBook.resetData(listEntryBook);
     }
 
     @Override
-    public ReadOnlyEntryBook getEntryBook() {
-        return entryBook;
+    public ReadOnlyEntryBook getListEntryBook() {
+        return listEntryBook;
     }
 
     @Override
     public boolean hasEntry(Entry entry) {
         requireNonNull(entry);
-        return entryBook.hasPerson(entry);
+        return listEntryBook.hasPerson(entry);
     }
 
     @Override
     public void deleteEntry(Entry target) {
-        entryBook.removePerson(target);
+        listEntryBook.removePerson(target);
     }
 
     @Override
     public void addEntry(Entry entry) {
-        entryBook.addPerson(entry);
+        listEntryBook.addEntry(entry);
         try {
             byte[] articleContent = Network.fetchAsBytes(entry.getLink().value);
             storage.addArticle(entry.getLink().value, articleContent);
         } catch (IOException ioe) {
             // Do nothing if we fail to fetch the page.
         }
-        updateFilteredEntryList(PREDICATE_SHOW_ALL_PERSONS);
+        updateFilteredEntryList(PREDICATE_SHOW_ALL_ENTRIES);
     }
 
     @Override
     public void setEntry(Entry target, Entry editedEntry) {
         requireAllNonNull(target, editedEntry);
 
-        entryBook.setPerson(target, editedEntry);
+        listEntryBook.setPerson(target, editedEntry);
     }
 
     @Override
     public void clearEntryBook() {
-        entryBook.clear();
+        listEntryBook.clear();
+    }
+
+    @Override
+    public void displayEntryBook(ReadOnlyEntryBook entryBook) {
```

Comment:
Does this mean any `Command` can display any `EntryBook`?

Reply:
`Model` has a fairly transparent API. A `Command` can even do `model.setListEntryBook(…)` (previously named `setEntryBook`) if it wants to which made me think that this is fine.